packages/synapse-react-client/test/lib/containers/table/datasets/DatasetItemsEditor.test.tsx

```diff
@@ -588,7 +593,8 @@ describe('Dataset Items Editor tests', () => {
       await clickCancel()
 
       // Verify that the warning dialog appears
-      expect(screen.queryByRole('dialog')).toBeInTheDocument()
+      const dialog = screen.queryByRole('dialog')
+      expect(dialog).toBeInTheDocument()
```

Comment:
Let's improve the original code; `getBy*` is the same as `queryBy*`, and also implicitly asserts that the element exists
```suggestion
      const dialog = screen.getByRole('dialog')
```